### PR TITLE
[OC-1089] Add perimeter management to admin screen

### PR DESCRIPTION
**Pull request by AlexGuironnetRTE**
A possible improvement may be to limit the state options in dropdowns to remove those that have already been selected to avoid duplicates, but I don't know if it's worth the complexity.

I also created OC-1504 to refactor the modals to avoid code duplication (currently it's causing the quality gate to fail because it now shows up as new code).

In release notes, under features:
[OC-1089] Add perimeter management to admin screen

[OC-1089]: https://opfab.atlassian.net/browse/OC-1089

### CODE REVIEW

File: ui/main/src/environments/environment.ts [hunk 1]
```diff
@@ -23,12 +23,14 @@ export const environment = {
...
         groups: 'http://localhost:2002/users/groups',
         entities: 'http://localhost:2002/users/entities',
+        perimeters: 'http://localhost:2002/users/perimeters',
         archives: '',
         processes: 'http://localhost:2002/businessconfig/processes',
         processGroups: 'http://localhost:2002/businessconfig/processgroups',
         config: 'http://localhost:2002/config/web-ui.json',
         menuConfig: 'http://localhost:2002/config/ui-menu.json',
-        time: 'http://localhost:2002/time'
+        time: 'http://localhost:2002/time',
```
Comment (freddidierRTE) [inline ui/main/src/environments/environment.ts:32]: remove this unused time url

**Comment by freddidierRTE** [comment]
Impossible to add new value in perimeter 

image

**Comment by freddidierRTE** [comment]
image

Either  :
- larger modal 
- or larger process multi-select and smaller state multi-select

File: ui/main/src/app/modules/admin/components/editmodal/perimeters/edit-perimeter-modal.component.ts [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+    this.initProcessOptions();
+    this.changeStatesWhenSelectProcess();
+
+    if (this.row) { // If the modal is used for edition, initialize the modal with current data from this row
+      const {id, process, stateRights} = this.row;
+      this.perimeterForm.patchValue({id, process}, { onlySelf: false });
+      const stateRightsControls = this.formBuilder.array(
+          stateRights.map((stateRight: StateRight) => EditPerimeterModalComponent.createStateRightFormGroup(stateRight.state, stateRight.right)));
+      this.perimeterForm.setControl('stateRights', stateRightsControls);
+      console.log('AGU after patch ', this.perimeterForm);
```
Comment (freddidierRTE) [inline ui/main/src/app/modules/admin/components/editmodal/perimeters/edit-perimeter-modal.component.ts:63]: Remove this console.log

Comment (AlexGuironnetRTE) [reply ui/main/src/app/modules/admin/components/editmodal/perimeters/edit-perimeter-modal.component.ts:63]: Done

File: ui/main/src/environments/environment.ts (hunk 1 above)
Comment (AlexGuironnetRTE) [reply ui/main/src/environments/environment.ts:32]: Done

**Pull request merged**